src/ds++/Release.cc

```diff
@@ -112,7 +112,7 @@ const std::string author_list(bool const use_doxygen_formatting) {
   current_developers.insert(fomdev(127, "Andrew T. Till"));
   current_developers.insert(fomdev(91, "Ondrej Certik"));
   current_developers.insert(fomdev(15, "David A. Dixon"));
-  current_developers.insert(fomdev(4, "Howard Pritchard"));
+  current_developers.insert(fomdev(5, "Howard Pritchard"));
```

Comment:
I'll check this out from develop to get rid of this issue.... I think I might have pulled from "master" at some point (rather than develop) which might explain some of this weirdness.